src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,97 @@
-export const Pagination = () => {};
+import { getNumbers, getPaginatedItems } from '../../utils';
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  setCurrentPage: (arg: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  setCurrentPage,
+}: Props) => {
+  const pages = getNumbers(1, Math.ceil(total / perPage));
+  const items = getNumbers(1, total).map(n => `Item ${n}`);
+  const paginatedItems = getPaginatedItems(items, perPage);
```

Comment:
The `items` and `paginatedItems` arrays seem to be used for displaying items, which might not be the responsibility of the Pagination component. The Pagination component should focus on handling the pagination logic and emitting the current page number, while the parent component should handle the display of items based on the current page.